java/google/registry/model/domain/rgp/GracePeriodStatus.java

```diff
@@ -90,6 +97,23 @@
    */
   TRANSFER("transferPeriod");
 
+  /**
+   * Provide a quick lookup of GracePeriodStatus from xml name.
+   */
+  private static final Map<String, GracePeriodStatus> XML_NAME_TO_GRACE_PERIOD_STATUS =
+      Maps.uniqueIndex(Iterables.filter(Arrays.asList(GracePeriodStatus.values()),
+          new Predicate<GracePeriodStatus>() {
```

Comment:
FYI I changed this to use Predicates.not(Predicates.equalTo(SUNRUSH_ADD)).  It's simpler.